README.md

```diff
@@ -49,7 +49,9 @@ var isEnabled = normalize(['string', 'boolean'], true);
 // Provide a function as first argument to do custom coercion
 var now = new Date();
 var enabledSince = normalize(function(value) {
-  return value.constructor === Date ? value : null;
+  if (value.constructor === Date) {
```

Comment:
Why not just `return value.constructor === Date ? value : undefined;` ? And come to think of it, wouldn't `instanceof Date` be better in case someone extends that class?